### ISSUE TITLE
Bug: Please there is issue on the time and date guide in codeigniter 4

### ISSUE THREAD

**Issue by ChibuezeAgwuDennis** [codeigniter4/CodeIgniter4#2351]
Check this to update:

[toDateString()](https://codeigniter4.github.io/userguide/libraries/time.html#todatestring)

Displays just the date portion of the Time:
```php
$time = Time::parse('March 9, 2016 12:00:00', 'America/Chicago');
echo $time->toDateTimeString();     // 2016-03-09
```
Change it to the function toDateString()
```php
$time = Time::parse('March 9, 2016 12:00:00', 'America/Chicago');
echo $time->toDateString();     // 2016-03-09
```

**Comment by musmanikram** [comment]
New into contribution. Picking this one as my first contribution

**Comment by jim-parry** [comment]
@musmanikram Make sure to read the contributing section in the repo, and welcome!

**Comment by jim-parry** [comment]
closed by #2352
